spec/user_identity_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe AppleSignIn::UserIdentity do
+  let(:jwt_sub) { user_identity }
+  let(:jwt_iss) { 'https://appleid.apple.com' }
+  let(:jwt_aud) { 'com.apple_sign_in' }
+  let(:jwt_iat) { Time.now.to_i }
+  let(:jwt_exp) { jwt_iat + 5.minutes }
+  let(:private_key) { OpenSSL::PKey::RSA.generate(2048) }
+  let(:jwk) { JWT::JWK.new(private_key) }
+  let(:jwt) do
+    {
+      iss: jwt_iss,
+      aud: jwt_aud,
+      exp: jwt_exp,
+      iat: jwt_iat,
+      sub: jwt_sub,
+      email: 'timothy@rootstrap.com',
```

Comment:
Not a good idea to use personal emails in open source projects. You don't know where the code will end 😆

Reply:
But my email is in the gemspec 🤷‍♂️ and i think it's not a personal email.